module/VuFind/src/VuFind/Config/AccountCapabilities.php

```diff
@@ -149,6 +149,17 @@ public function getListTagSetting()
         return $this->config->Social->listTags ?? 'disabled';
     }
 
+    /**
+     * Get scheduled search setting ('enabled' or 'disabled').
+     *
+     * @return string
+     */
+    public function getScheduledSearchSetting()
```

Comment:
I know the other methods return 'enabled' or 'disabled', but could this still be more alike `isAccountAvailable` so that it returns a bool?

Reply:
Good point. I've simplified it.